packages/api3-voting/contracts/Api3Voting.sol

```diff
@@ -97,6 +97,7 @@ contract Api3Voting is IForwarder, AragonApp {
         voteTime = _voteTime;
         // The pool acts as the MiniMe token
         api3Pool = IApi3Pool(_token);
+        require(_voteTime <= api3Pool.EPOCH_LENGTH(), ERROR_VOTING_LENGTH);
```

Comment:
Should be exactly equal

Reply:
Then I think we shouldn't create this variable anyway, it will be the same for all different votes

Reply:
Will this refactoring be ok?

Reply:
I just wanted to keep the `initialize()` interface identical to the original `Voting.sol`.

Ideally we should just go
```
voteTime = api3Pool.EPOCH_LENGTH();
```
Is there a reason why we can't do this?

Reply:
We can do this, do you want to leave _voteTime as a parameter that we are passing? And do we need to check it if you want to?